tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -27,6 +29,7 @@ import (
     "../wski18n"
 
     "github.com/fatih/color"
+    "github.com/ghodss/yaml"
```

Comment:
New dependency looks like is Apache friendly MIT https://github.com/ghodss/yaml/blob/master/LICENSE